src/Components/Book/Book.js

```diff
@@ -1,2 +1,17 @@
 import React from 'react';
-import Book from './Book.css'
\ No newline at end of file
+import ReactDOM from 'react-dom';
+import './Book.css'
+
+
+const Book = ({book}) => {
+  return (
+    <section class="book-card-section">
+        <h1>{book.title}</h1>
+        <h3>{book.author}</h3>
+        <h3>Ranking: {book.rank}</h3>
+        <img className="Book-card-image" alt="Book cover" src={book.book_image} />
```

Comment:
Is there a way we can make the alt text dynamic so that each book cover has unique alt text?

Reply:
@NicoleLatifi great idea!!! Are you thinking we add the book title to the name as well?